drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/RightPanelViewImpl.html

```diff
@@ -100,8 +100,31 @@
         <!-- content for cheatsheet tab -->
         <div id="kieScenarioCheatsheetTabContent"
              class="tab-pane fade"
-             aria-labelledby="Scenario Cheatsheet">
-            <p>Create a row for each scenario, and input values using the following syntax.</p>
+             aria-labelledby="Scenario Cheatsheet" style="padding: 1em">
+            <p>Use contextual menu to add/remove columns and rows</p>
+
+            <p>Left click on the header of a column and then edit the mapping with a field through the right panel</p>
+
+            <p>Double click on the field to set the mapping</p>
+
+            <p>Each cell can contain an expression using the following syntax</p>
+
+            <ul>
+                <li><i>= value (or no symbol)</i>: to specify equals to a value. This is the only operator supported under GIVEN section and it is the default</li>
```

Comment:
Don't start the description with "to specify ...". 
Change line 113 to >> **specifies** equals to a value. This is the **default and only** operator that is supported under the Given section.